guardrails/utils/openai_utils/v0.py

```diff
@@ -16,7 +16,7 @@
 
 
 def get_static_openai_create_func():
-    return openai.Completion.create
+    return openai.completions.create
```

Comment:
These shouldn't change.  They are here for backwards compatibility with v0.x of the openai library.